.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ orbs:
 references:
   build_job: &build
     docker:
-      - image: circleci/golang:1.13-stretch
+      - image: circleci/golang:1.14-stretch
```

Comment:
I think we can use [setting-an-environment-variable-in-a-project](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project) feature in Circle CI, merge all redundant values for golang Docker image to one place.

This means single var for all jobs :D  any thoughts?

Reply:
I think it is best to have it set in the Makefile, since these CircleCI images might not match, and we need the version set when we build the images locally during development.